README.md

```diff
@@ -264,11 +266,65 @@ The recommended strategy for Heroku is to setup a
 *:information_desk_person: Note: for high volume apps Heroku log drains will drop messages. This
 is true for any Heroku app, in which case we recommend the Network method below.*
 
+---
+
 </p></details>
 
 <details><summary><strong>All other platforms (Network / HTTP)</strong></summary><p>
 
-Coming soon!
+Timber does *not* force an HTTP client on you. The following instruction utilize the Timber default
+`Timber.Transports.HTTP.HackneyClient`. This is a highly efficient client that utilizes hackney,
+batching, stay alive connections, connection pools, and msgpack to deliver logs with high
+throughput and little overhead. If you'd like to use another client see
+`Timber.Transports.HTTP.Client`.
+
+1. *Add* HTTP dependencies to `mix.exs`:
+
+  ```elixir
+  def application do
+    [applications: [:hackney, :timber]] # <-- Be sure to add hackney!
+  end
+
+  def deps do
+    [
+      {:timber, "~> 1.0"},
+      {:hackney, "~> 1.6"} # <-- ADD ME
+    ]
+  end
+  ```
+
+2. *Configure* Timber to use the Network transport in `config/prod.exs`:
+
+  ```elixir
+  # config/prod.exs
+
+  config :timber,
+    transport: Timber.Transports.Network,
+    api_key: System.get_env("TIMBER_API_KEY")
```

Comment:
Sure, I felt the API key had potential to be used elsewhere in the future. I also noticed we've been using generic keys, which I agree with, as it doesn't tie our user's configuration to exact modules. Ex:

```
config :timber, :timber_http_transport, ...
```